pyanaconda/iutil.py

```diff
@@ -1242,3 +1242,10 @@ def touch(file_path):
     # even when the path points to dirrectory
     if not os.path.exists(file_path):
         os.mknod(file_path)
+
+def firstNotNone(*args):
+    """Return the first argument that is not None."""
+    for arg in args:
+        if arg is not None:
+            return arg
```

Comment:
I've seen this implementation before, but it is really hard to read. What are the advantages?